CardForm/src/test/java/com/braintreepayments/cardform/view/CvvEditTextTest.java

```diff
@@ -70,8 +70,6 @@ public void hintChangesForCardType() {
 
             assertEquals(RuntimeEnvironment.application.getString(cardType.getSecurityCodeName()),
                     mView.getTextInputLayoutParent().getHint());
-            assertEquals(RuntimeEnvironment.application.getString(cardType.getSecurityCodeName()),
```

Comment:
This assertion should have been removed in the accessibility update PR (since we removed the content description), removing it now to fix CI.